ExampleMod/Backgrounds/ExampleCloud.cs

```diff
@@ -0,0 +1,59 @@
+﻿using Microsoft.Xna.Framework;
+using System;
+using Terraria;
+using Terraria.DataStructures;
+using Terraria.Utilities;
+using static Terraria.Cloud;
+using static Terraria.WorldGen;

```

Comment:
`using static` makes the code quite confusing

Reply:
I do agree, in some scenarios it's fine though. This file is so small it really doesn't matter.

Reply:
for consistency with example mod I'm fine with no using statics, because it's not a necessary thing to the function of the code and something that was introduced in like version 6 I believe, and a lot of programmers don't really use it.